developers/weaviate/model-providers/_includes/provider.connect.py

```diff
@@ -25,6 +25,10 @@
 # Recommended: save sensitive data as environment variables
 cohere_key = os.getenv("COHERE_APIKEY")
 # END CohereInstantiation
+# START FriendliAIInstantiation
```

Comment:
is it Friendli here? or FriendliAI?

Reply:
As those directives are just markers that are not publicly opened to viewers, we can change them to `Friendli`.
But I feel like `FriendliAI` is a bit more natural as a prefix to the word 'Instantiation'

Reply:
Since it is not public, we should more strictly follow the convention. Please change the values to Friendli